docker-compose.yaml

```diff
@@ -38,12 +38,31 @@ services:
       MINIO_AUDIT_WEBHOOK_ENDPOINT_target1: http://cwm-worker-logger:8500/logs
       MINIO_GATEWAY_DEPLOYMENT_ID: "docker-compose-http"
       MINIO_API_SELECT_PARQUET: "${MINIO_API_SELECT_PARQUET}"
+      MINIO_KMS_AUTO_ENCRYPTION: "off"
+      MINIO_KMS_KES_ENDPOINT: "https://minio-kes:7373"
+      MINIO_KMS_KES_CERT_FILE: "/kes/certs/minio.cert"
+      MINIO_KMS_KES_KEY_FILE: "/kes/certs/minio.key"
+      MINIO_KMS_KES_CAPATH: "/kes/certs/server.cert"
+      MINIO_KMS_KES_KEY_NAME: minio-encryption-key
     volumes:
       - "storage:/storage"
       - "cache:/cache"
+      - "minio-kes-certs:/kes/certs"
     depends_on:
+      - minio-kes
       - cwm-worker-logger
 
+  minio-kes:
```

Comment:
please add a separate docker-compose config for the minio-kes setup, like we do for gateway - https://github.com/CloudWebManage/cwm-worker-deployment-minio/blob/main/README.md#gateway-to-other-minio-instance

Reply:
also, document how to use it in the README